crates/pera-core/src/consensus_handler.rs

```diff
@@ -371,25 +372,30 @@ impl<C: CheckpointServiceNotify + Send + Sync> ConsensusHandler<C> {
                             authority_index, session_id
                         );
 
-                        let dwallet_mpc_manager = self.epoch_store.dwallet_mpc_manager.get();
-                        let output_verification_result = match dwallet_mpc_manager {
-                            Some(mpc_manager) => {
-                                let mut dwallet_mpc_manager = mpc_manager.lock().await;
-                                match dwallet_mpc_manager.try_verify_output(value, session_id) {
-                                    Ok(is_valid) => is_valid,
-                                    Err(e) => {
-                                        error!(
-                                                    "Error verifying DWalletMPCOutput output from session {:?} and party {:?}: {:?}",
-                                                    session_id, authority_index, e
-                                                );
-                                        OutputVerificationResult::Malicious
-                                    }
-                                }
-                            }
-                            None => {
-                                // TODO (#250): Make sure that the MPC manager is initialized before MPC events emitted.
-                                error!("MPC manager was not initialized when verifying DWalletMPCOutput output from session {:?}", session_id);
-                                OutputVerificationResult::Duplicate
+                        let Some(dwallet_mpc_manager) = self.epoch_store.dwallet_mpc_manager.get()
+                        else {
+                            error!("MPC manager was not initialized when verifying DWalletMPCOutput output from session {:?}", session_id);
+                            return;
+                        };
+                        let mut dwallet_mpc_manager = dwallet_mpc_manager.lock().await;
+                        if dwallet_mpc_manager.malicious_actors.contains(
+                            &self.committee
+                                .authority_pubkey_by_index(authority_index)
+                                .unwrap(),
```

Comment:
Safe to unwrap here as it's being done by Sui [here](https://github.com/dwallet-labs/dwallet-network/blob/29929ded135f05578b6ce33b52e6ff5e894d0487/crates/pera-core/src/consensus_handler.rs#L406-L409)